pages/search.js

```diff
@@ -25,10 +24,12 @@ import StyledInput from '../components/StyledInput';
 import StyledLink from '../components/StyledLink';
 import { H1, P } from '../components/Text';
 
+import defaultColors from '../lib/theme/colors';
+
 const SearchInput = styled(StyledInput)`
   &&& {
     border: none;
-    border-bottom: 2px solid ${colors.blue};
+    border-bottom: 2px solid ${defaultColors.blue};
```

Comment:
Not sure about how this reference should be made. Appears to be a parameter within a styled-component definition.

Please advise--thx!

Reply:
@ericyoungson : I think this can be achieved in two ways; you could use the `themeGet` function of `styled-system`, something like;

`border-bottom: 2px solid ${themeGet('colors.red.600')};`

or you could use props, something like;

border-bottom: 2px solid ${props => props.theme.colors.red[400]}

Personally, I would use the second approach since it doesn't use the unnecessary import. Hope that helps. 😄 

(@Betree : please add if I am missing anything) 😸

Reply:
Yes, @SudharakaP I was thinking something like the second approach. Apologies for the delay; I'm just getting back to this.

Reply:
@ericyoungson: No worries, let us know if you need any further help. 😄